includes/AMP/Sanitization.php

```diff
@@ -26,18 +26,9 @@
 
 namespace Google\Web_Stories\AMP;
 
-use Google\Web_Stories_Dependencies\AMP_Allowed_Tags_Generated;
-use Google\Web_Stories_Dependencies\AMP_Content_Sanitizer;
-use Google\Web_Stories_Dependencies\AMP_DOM_Utils;
-use Google\Web_Stories_Dependencies\AMP_Layout_Sanitizer;
-use Google\Web_Stories_Dependencies\AMP_Script_Sanitizer;
-use Google\Web_Stories_Dependencies\AMP_Style_Sanitizer;
-use Google\Web_Stories_Dependencies\AMP_Tag_And_Attribute_Sanitizer;
-use Google\Web_Stories_Dependencies\AmpProject\Amp;
-use Google\Web_Stories_Dependencies\AmpProject\Attribute;
+use Google\Web_Stories_Dependencies\{AMP_Allowed_Tags_Generated,AMP_Content_Sanitizer,AMP_DOM_Utils,AMP_Layout_Sanitizer,AMP_Script_Sanitizer,AMP_Style_Sanitizer,AMP_Tag_And_Attribute_Sanitizer};
+use Google\Web_Stories_Dependencies\AmpProject\{Amp,Attribute,Extension,Tag};
 use Google\Web_Stories_Dependencies\AmpProject\Dom\Document;
-use Google\Web_Stories_Dependencies\AmpProject\Extension;
-use Google\Web_Stories_Dependencies\AmpProject\Tag;
```

Comment:
I'm not a huge fan of grouped `use` declarations as I don't think they add much value and can actually _decrease_ readability. This change here is a great example.

WDYT?

Reply:
This is the only example of where I don't love it. I can change it back if you want. I tweaks these manually, so I wish I knew before I did it.

Reply:
There's probably an automated way to add these using PHPCS.

I don't mind keeping it now that you've done the work anyway. Just don't want to enforce this style.

Reply:
I dont like them, I have reverted the change.